frontend/javascripts/dashboard/advanced_dataset/dataset_action_view.tsx

```diff
@@ -330,7 +236,7 @@ export function getDatasetActionContextMenu({
             },
           }
         : null,
-      dataset.isEditable && dataset.isActive
```

Comment:
Am I correct, that we no longer have a "not imported" state for a dataset? In that case, is it possible for a dataset to not be active? The only option would be to delete a dataset :thinking: 

My thought is that we may not need this field `isActive` anymore / do not need to consider it any longer in the frontend :thinking:

Reply:
> Am I correct, that we no longer have a "not imported" state for a dataset? In that case, is it possible for a dataset to not be active?

I don't know what happens now if a datasource properties json does not exist (or is incomplete). I assume the dataset won't be active then. However, the front-end should still show the edit button. Otherwise, the dataset would not be usable/fixable as you say.